api/admin/controller.py

```diff
@@ -168,6 +170,24 @@ def details(self, data_source, identifier):
         return entry_response(
             AcquisitionFeed.single_entry(self._db, work, annotator)
         )
+        
+    def complaints(self, data_source, identifier):
+        """Return detailed complaint information for admins."""
+        
+        pool = self.load_licensepool(data_source, identifier)
+        if isinstance(pool, ProblemDetail):
+            return pool
+        work = pool.work
+        counter = self._count_complaints_for_work(work)
+        response = dict({
+            "book": { 
```

Comment:
I use "book" in code comments as a kind of shorthand, but in code and data I try not to use that term, since the 'book' here could just as easily be an audiobook or borrowable video. Can you use a more generic term, or even an internal term like 'license_pool'?